interface-definitions/firewall.xml.in

```diff
@@ -205,6 +205,63 @@
               </leafNode>
             </children>
           </tagNode>
+          <tagNode name="lists">
```

Comment:
Well they aren't always remote, though external could work, remember that it works with local on the filesystem as well as remote.

Reply:
Before I change this part I'd like to make sure everyone is in agreement, since this will take to most refactoring and file renaming.

I'm okay with `external-list`, so it would be `set firewall group external-list <listname>` so I'll convert all occurrences of `lists` or `list` to `external-list`

Reply:
Changed to `external-list`